package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "vc-holder-http",
+  "version": "0.0.0-1",
```

```suggestion
  "version": "0.0..1",
```

0.0.0 does not work well with the release tooling.

---

@mattcollier thank you for pointing that out, we came across that issue just yesterday.